src/controllers/expenses.js

```diff
@@ -0,0 +1,119 @@
+import * as expenseService from '../services/expenses.js';
+import * as userService from '../services/users.js';
+
+export async function getAll(req, res) {
+  const searchQuery = req.query;
+
+  const expenses = await expenseService.getAll(searchQuery);
+
+  res.send(expenses);
+}
+
+export async function getOne(req, res) {
+  const { expenseId } = req.params;
+
+  const foundExpense = await expenseService.getById(expenseId);
+
+  if (!foundExpense) {
+    res.statusCode = 404;
+    res.end(`Expense with id ${expenseId} was not found`);
+
+    return;
+  }
+
+  res.send(foundExpense);
+}
+
+export async function add(req, res) {
+  const expenseData = req.body;
+
+  const expectedFields = [
+    'userId',
+    'spentAt',
+    'title',
+    'amount',
+    'category',
+    'note',
+  ];
```

Comment:
you can get them from model
```suggestion
  const expectedFields = Expense.getAttributes();
```